### ISSUE TITLE
Worse outputs compared to older versions

### ISSUE THREAD

**Issue by keezinv2** [bmaltais/kohya_ss#1186]
Hi there!

I have been playing around with lora for a bit already, but I noticed that I cannot update to the newest versions because as soon as I do the results become incredibly bad. I am unsure why that happens, I have reinstalled the entire GUI from scratch, I changed from Torch 1 to 2, Fp to Bf etc. Nothing helps.

The version that currently works for me and is producing optimal results is 21.6.6.
I tried multiple different versions and all of them produce total mess (for me).
Maybe I am skipping something that was not required during the 21.6.6 and prior older versions of the installation process.

Could you please help me, I am stuck I tried all that came to my mind.

![00000-2823710171](https://github.com/bmaltais/kohya_ss/assets/139295011/6c10deb9-dadc-4405-94e2-e9be2ce52b2a)
(this is an example of what I get when trying to use the newest versions)

PS. I use all the same settings as I did prior with older versions and I followed the tutorial on YT.

**Comment by aimerchichi** [comment]
i have the same problem with worse results

**Comment by Jackfritt** [comment]
Same here it worked once month ago, and i followed latest YT tutorials. Reinstall doesnt work either. Maybe I try to install an rly old version..

**Comment by keezinv2** [comment]
Ye the version from around 1-2 months ago works perfectly normal, that's why im thinking if maybe there is something odd with the instalation

**Comment by Jackfritt** [comment]
Goin back to 21.6.6 or older, as proposed is a problem too. Cant fill out fields in webbrowser. I get an error message in all fields with older Version. Habe to investigate why…

**Comment by bmaltais** [comment]
Try deleting the bench when you backrev and re-run setup. Hope this help.

**Comment by Jackfritt** [comment]
I also tried it and solved the problem with the file path inputs. There is a workaround for this, but then after training starts the caching is really slow and when it begins training i get another new error message. Will try to investigate into this. Seems there are a lot of older depends that dont work with never drivers or software. But first im 2 weeks off..

**Comment by keezinv2** [comment]
> Try deleting the bench when you backrev and re-run setup. Hope this help.

sorry but could you specify a bit more? I'm sorry I am kinda new to all of this stuff, of course if you can

**Comment by thijsi123** [comment]
May I ask which version worked best for you (anyone may answer)?
I reinstalled my install from around 2 months ago and I'm regretting it big time!!!

**Comment by aimerchichi** [comment]
i think i solved it, try lowering training steps, i think the new version has stronger learning rate, so with old setting seems always to be overtrained

**Comment by keezinv2** [comment]
> i think i figured it out, try lowering training steps, i think the new version has stronger learning rate, so with old setting seems always to be overtrained

how many steps do you use now? could you let me know?
